tox_poetry_installer/hooks.py

```diff
@@ -165,51 +90,40 @@ def tox_testenv_install_deps(venv: ToxVirtualEnv, action: ToxAction) -> Optional
     :param venv: Tox virtual environment object with configuration for the local Tox environment.
     :param action: Tox action object
     """
-
     try:
-        poetry = utilities.check_preconditions(venv, action)
+        poetry = utilities.check_preconditions(tox_env)
     except exceptions.SkipEnvironment as err:
         if isinstance(err, exceptions.PoetryNotInstalledError) and (
-            venv.envconfig.config.option.require_poetry or venv.envconfig.require_poetry
+            tox_env.core["require_poetry"] or tox_env.conf["require_poetry"]
```

Comment:
This looks like it raises a `KeyError` since the `require_poetry` option isn't set on the core config object. Since we're only setting this option this per-environment we can drop the `tox_env.core["require_poetry"]` clause.

Reply:
Sounds good :+1: 
I have removed the core check